academic-services/knowledge-exploration-service/index.yml

```diff
@@ -1,39 +1,65 @@
+
 metadata:
-  title: MAKES documentation
-  description: Learn about Microsoft Academic Knowledge Exporation Service, including technology overviews, API references, how-to guides, tutorials and more.
-  meta.description: Learn about Microsoft Academic Knowledge Exporation Service, including technology overviews, API references, how-to guides, tutorials and more.
+  title: Microsoft Academic Knowledge Exploration Service
+  description: Learn about Microsoft Academic Knowledge Exploration Service, including technology overviews, API references, how-to guides, tutorials and more.
   ms.topic: landing-page
-  ms.date: 2020-02-10
-abstract:
-  description: Microsoft Academic Knowledge Exploration Service offers a fast and effective way to add interactive academic search and refinement to applications. Learn about Microsoft Academic Knowledge Exporation Service, including technology overviews, API references, how-to guides, tutorials and more.
-sections:
-- title: Technology overview
-  items: 
-  - type: list
-    style: unordered
-    items:
-    - html: <a href="introduction">About Microsoft Academic Knowledge Exploration Service (MAKES)</a>
-- title: Get started
-  items:
-  - type: paragraph
-    text: Learn how to deploy, manage, and use MAKES.
-  - type: list
-    style: unordered
-    items:
-    - html: <a href="get-started-setup-provisioning">Sign up for Microsoft Academic Knowledge Exploration Service (MAKES)</a>
-    - html: <a href="get-started-create-api-instances">Deploy a basic MAKES instance using the provided scripts</a>
-- title: How to
-  items:
-  - type: paragraph
-    text: Learn how to upgrade and manage MAKES
-  - type: list
-    style: unordered
-    items:
-    - html: <a href="how-to-deploy-new-indexes-to-makes">How to deploy new releases to an existing instance of MAKES</a>
-    - html: <a href="how-to-delete-api-instances">How to delete a MAKES deployment</a>
-    - html: <a href="how-to-create-index-from-mag">Create custom MAKES index from a MAG release</a>
-    
-    
\ No newline at end of file
+  author: DarrinEide
+  ms.author: darrine
+  ms.date: 7/31/2020
+
+landingContent:
+  - title: About MAKES
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: What is MAKES?
+            url: introduction.md
+      - linkListType: concept
+        links:
+          - text: How MAKES understands natural language queries
+            url: concept-queries.md
+
+  - title: Get started
+    linkLists:
+      - linkListType: get-started
+        links:
+          - text: Sign up for MAKES
+            url: get-started-setup-provisioning.md
+      - linkListType: how-to-guide
+        links:
+          - text: Deploy new MAKES instance
```

Comment:
>Deploy new MAKES instance [](start = 18, length = 25)

Let's modify "Create an API instance" to "Deploy new MAKES instance" on the main navigation rail to be consistent.